providers/netcup/api.go

```diff
@@ -13,9 +13,7 @@ const (
 )
 
 type netcupProvider struct {
-	domainIndex      map[string]string
```

Comment:
These might be generated from another source.  Leave in.

Reply:
Struct fields was also commented out